lib/galaxy/webapps/galaxy/controllers/admin.py

```diff
@@ -136,22 +136,26 @@ def get_value(self, trans, grid, user):
     ]
     operations = [
         grids.GridOperation("Manage Information",
-                            condition=(lambda item: not item.deleted),
+                            condition=(lambda item, app: not item.deleted),
                             allow_multiple=False,
                             url_args=dict(controller="user", action="information", webapp="galaxy")),
         grids.GridOperation("Manage Roles and Groups",
-                            condition=(lambda item: not item.deleted),
+                            condition=(lambda item, app: not item.deleted),
                             allow_multiple=False,
                             url_args=dict(action="form/manage_roles_and_groups_for_user")),
         grids.GridOperation("Reset Password",
-                            condition=(lambda item: not item.deleted),
+                            condition=(lambda item, app: not item.deleted),
                             allow_multiple=True,
                             url_args=dict(action="form/reset_user_password"),
                             target="top"),
         grids.GridOperation("Recalculate Disk Usage",
-                            condition=(lambda item: not item.deleted),
+                            condition=(lambda item, app: not item.deleted),
                             allow_multiple=False),
         grids.GridOperation("Generate New API Key",
+                            allow_multiple=False,
+                            async_compatible=True),
+        grids.GridOperation("Activate User",
+                            condition=(lambda item, app: app.config.user_activation_on and not item.active),
```

Comment:
I would suggest not adding the option in this list, swapping the lambda back to a single argument without `app`, and then just conditionally add the GridOperation when the grid is initialized, based on the config at hand.